.github/workflows/code-quality.yml

```
@@ -49,17 +75,77 @@ jobs:
+          git config --local user.name "github-actions[bot]"
+          
+          # Run auto-fix
+          pnpm biome check --write . || true
+          pnpm prettier:yaml || true
+          
+          # Check if there are changes
+          if [[ -n $(git status -s) ]]; then
+            echo "📝 Found auto-fixable issues, committing fixes..."
+            git add .
+            git commit -m "🤖 Auto-fix: Format code with Biome and Prettier
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)